autorelease/github.py

```diff
@@ -89,27 +89,28 @@ def list_org_repos(self, org: str, type: str = None) -> Generator[dict, None, No
             url = response.links.get("next", {}).get("url")
 
     def list_org_issues(
-        self, org: str, filter: str = None, state: str = None, labels: str = None
+        self, org: str, state: str = None, labels: str = None
     ) -> Generator[dict, None, None]:
-        url = f"{self.GITHUB_ROOT}/orgs/{org}/issues"
-
-        # GitHub sometimes returns 5xx errors for this request.
-        # Retry after 500, 502 response up to 4 times.
-        max_retries = Retry(status=4, status_forcelist=[500, 502])
-        adapter = requests.adapters.HTTPAdapter(max_retries=max_retries)
-        self.session.mount(url, adapter)
-
-        while url:
-            response = self.session.get(
-                url, params=dict(filter=filter, state=state, labels=labels)
-            )
-            response.raise_for_status()
-            if response.status_code >= 400:
-                logging.error(response.text)
-            for item in response.json():
-                yield item
-
-            url = response.links.get("next", {}).get("url")
+        for repo in self.list_org_repos(org, "public"):
```

Comment:
It may be covered elsewhere, but make sure we're filtering out archived repos as well :)